server/src/main/java/com/microsoft/java/bs/core/internal/utils/JavaUtils.java

```diff
@@ -0,0 +1,79 @@
+package com.microsoft.java.bs.core.internal.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.Runtime.Version;
+
+/**
+ * Utility class for Java.
+ */
+public class JavaUtils {
+
+  public JavaUtils() {
```

Comment:
should be private?

Reply:
Fixed in commit [04ed182](https://github.com/microsoft/build-server-for-gradle/pull/165/commits/04ed182c48bf04e766b4189900cdfbb470550b50)